addons/html_editor/static/src/others/embedded_components/plugins/excalidraw_plugin/excalidraw_dialog/excalidraw_dialog.js

```diff
@@ -18,7 +18,7 @@ export class ExcalidrawDialog extends Component {
 
     setup() {
         super.setup();
-        this.state = useState({});
+        this.state = useState({ isInsertionDisabled: true });
```

Comment:
No need to add a state value which meaning is almost the same as `hasError`. Instead I suggest adding the `urlValue` in the state:
```suggestion
        this.state = useState({
            hasError: false,
            urlValue: "",
        });
```

`onKeyDown` should be refactored like so, it should not reset the error state:
```js
    onKeyDown(event) {
        if (event.key === "Enter") {
            this.saveURL();
        }
    }
```